ig/input/fsh/nbs-orc-segment.fsh

```diff
@@ -2,15 +2,20 @@ Logical: ORCSegment
 Id: segment-orc-logical-model
 Title: "ORC Segment"
 Description: "The ORC Segment"
-* orderControl 1..1 SU string "ORC-1 Order Control"
-* ORCPlacerOrderNumber 1..1 SU string "ORC-2 Placer Order Number"
-* fillerOrderNumber 1..1 SU string "ORC-3 Filler Order Number"
-* placerGroupNumber 1..1 SU string "ORC-4 Placer Group Number"
-* dateTimeOfTransaction 1..1 SU string "ORC-9 Date/Time of Transaction"
-* orderingProvider 1..1 SU string "ORC-12 Ordering Provider"
-* orderingFacilityName 1..1 SU string "ORC-11 prdering Facility Name"
-// other optional fields
-
+* orderControl 1..1 SU id "ORC-1 Order Control"
```

Comment:
I would make this `code`.  [An `id`](http://hl7.org/fhir/R4/datatypes.html#primitive) is a...
> Any combination of upper- or lower-case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'), '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID or any other identifier pattern that meets these constraints.)

While that description does apply to this field, I feel that `code` is more appropriate.
> Indicates that the value is taken from a set of controlled strings defined elsewhere (see [Using codes](http://hl7.org/fhir/R4/terminologies.html) for further discussion). Technically, a code is restricted to a string which has at least one character and no leading or trailing whitespace, and where there is no whitespace other than single spaces in the contents

Specifically, the order control field is taken from the [0119 controlled set of strings](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0119).

```suggestion
* orderControl 1..1 SU code "ORC-1 Order Control"
```